car_custom_model.yaml

```diff
@@ -0,0 +1,6 @@
+speed_factor:
```

Comment:
NOTE: this is just an example of a custom model. Once we've decided on finalized knob settings per vehicle type I will update